rankers/SimpleAggregateRanker/__init__.py

```diff
@@ -38,4 +38,4 @@ def score(self, match_idx, query_chunk_meta, match_chunk_meta, *args, **kwargs):
                 raise ValueError(f'Setting "is_reversed_score" to True does not allow to have an aggregated document '
                                  f'negative score')
             aggregated_score = 1. / (1. + aggregated_score)
-        return aggregated_score
+        return self.get_doc_id(match_idx), aggregated_score
```

Comment:
rankers do not return ids anymore

Reply:
I'm getting rid of other changes and only keep the Bimatchranker. 😂 
That's why I propose to merge into your rather than master.

Reply:
as u want, but u can target master

Reply:
so just copy the master folder from SimpleAggregateRanker and handle that ranker in a different branch